service/index.js

```diff
@@ -0,0 +1,34 @@
+const Contact = require("./schemas/contact");
+
+const listContacts = async () => {
+  return Contact.find();
+};
+
+const getContactById = async (contactId) => {
+  return Contact.findOne({ _id: contactId });
+};
+
+const removeContact = async (contactId) => {
+  Contact.deleteOne({ _id: contactId });
```

Comment:
Widzę że tego nie zwracasz, a w jaki sposób się dowiadujesz czy kontakt został usunięty? Bo jeżeli go usuniesz tutaj i nie zwrócisz wyniku to jak funkcja nadrzędna wie co się wydarzyło?

Reply:
poprawione